docs/user/04-40-troubleshooting-istio-sidecar-version.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Incompatible Istio sidecar version after Kyma upgrade
+---
+
+## Symptom
+
+You upgraded Kyma and mesh connectivity is broken.
+
+## Cause
+
+By default, Kyma has sidecar injection disabled - there is no automatic sidecar injection into any Pod in a cluster. For more information, read the document about [enabling Istio sidecar proxy injection](../../operations/smsh-01-istio-enable-sidecar-injection.md).
+
+The sidecar version in Pods must match the installed Istio version. Otherwise, mesh connectivity may be broken.
+This issue may appear during Kyma upgrade. When Kyma is upgraded to a new version along with a new Istio version, existing sidecars injected into Pods remain in an original version.
+Kyma contains `istio-proxy-reset` <!--`istio-proxy-reset` is no longer a job. Update and explain what `istio-proxy-reset` actually is once Reconciller is ready to use.--> that performs a rollout for most common workload types, such as Deployments, DaemonSets, etc. The job ensures all Kyma components are properly updated.
```

Comment:
```suggestion
Istio module contains `ProxySidecarReconcilation` component that performs a rollout for most common workload types, such as Deployments, DaemonSets, etc. The job ensures all running sidecars are properly updated.
```

Reply:
> The job ensures all running sidecars are properly updated.

Is there still a job for that or is it also the `ProxySidecarReconcilation` component's responsibility?

Reply:
Correct, it should have been `component` not `job`